src/library_browser.js

```diff
@@ -1120,9 +1120,7 @@ var LibraryBrowser = {
   },
 
   emscripten_set_window_title__proxy: 'sync',
-  emscripten_set_window_title: (title) => {
-    setWindowTitle(UTF8ToString(title));
-  },
+  emscripten_set_window_title: (title) => document.title = UTF8ToString(title),
```

Comment:
Do we need a dep on `UTF8ToString` here?

Reply:
That missing dep is pre-existing, so this change should effect that.

I just checked and it looks like the reason this is OK is that we have `UTF8ToString` listed in the implicit deps list: https://github.com/emscripten-core/emscripten/blob/dfb3585e8f337cb010c7ed09a3942d436633217c/src/jsifier.js#L270-L276. 

So think we can leave it as is for now.